test/pco-license/CFAReclaimerFacet.ts

```diff
@@ -0,0 +1,275 @@
+        const claimPrice = await basePCOFacet
+        .connect(await ethers.getSigner(bidder))
+        .claimPrice();
+
+        const op1 = paymentToken.transfer({
+          receiver: bidder,
+          amount: claimPrice.add(totalCollateral).toString(),
+        });
+        await op1.exec(await ethers.getSigner(bidder));
+
+        // Allow spending of claimPrice
+        const op2 = paymentToken.approve({amount: claimPrice.add(totalCollateral), receiver: basePCOFacet.address});
+        await op2.exec(await ethers.getSigner(bidder));
+
+        // Approve flow creation
+        const op3 = ethersjsSf.cfaV1.updateFlowOperatorPermissions({
+          superToken: paymentToken.address,
+          flowOperator: basePCOFacet.address,
+          permissions: 1,
+          flowRateAllowance: contributionRate.toString(),
+        });
+        await op3.exec(await ethers.getSigner(bidder));
+
+        const txn = await basePCOFacet.connect(await ethers.getSigner(bidder)).claim(contributionRate, forSalePrice);
+        await txn.wait();
+        await expect(txn).to.emit(basePCOFacet, "LicenseReclaimed");
+      });
+
+      it("calls license.safeTransferFrom", async () => {
+        const { basePCOFacet, mockParamsStore, paymentToken, ethersjsSf, mockLicense } = await BaseFixtures.afterPayerDelete();
+        const { bidder, user } = await getNamedAccounts();
+
+        const contributionRate = BigNumber.from(100);
+        const forSalePrice = await rateToPurchasePrice(
+          mockParamsStore,
+          contributionRate
+        );
+        const requiredBuffer = await ethersjsSf.cfaV1.contract
+        .connect(await ethers.getSigner(bidder))
+        .getDepositRequiredForFlowRate(
+          paymentToken.address,
+          contributionRate
+        );
+        const totalCollateral = forSalePrice.add(requiredBuffer);
+
+        // Transfer payment token for buffer
+        const claimPrice = await basePCOFacet
+        .connect(await ethers.getSigner(bidder))
+        .claimPrice();
+
+        const op1 = paymentToken.transfer({
+          receiver: bidder,
+          amount: claimPrice.add(totalCollateral).toString(),
+        });
+        await op1.exec(await ethers.getSigner(bidder));
+
+        // Allow spending of claimPrice
+        const op2 = paymentToken.approve({amount: claimPrice.add(totalCollateral), receiver: basePCOFacet.address});
+        await op2.exec(await ethers.getSigner(bidder));
+
+        // Approve flow creation
+        const op3 = ethersjsSf.cfaV1.updateFlowOperatorPermissions({
+          superToken: paymentToken.address,
+          flowOperator: basePCOFacet.address,
+          permissions: 1,
+          flowRateAllowance: contributionRate.toString(),
+        });
+        await op3.exec(await ethers.getSigner(bidder));
+
+        const txn = await basePCOFacet.connect(await ethers.getSigner(bidder)).claim(contributionRate, forSalePrice);
+        await txn.wait();
+        expect(
```

Comment:
Can this be checked in the same test above? Is everything the same?

Reply:
This can be actually but just kept the way it was done in the old reclaimer tests, going to change that